<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EzConv</string>
  </property>
  <property name="windowIcon">
   <iconset theme="bitcoin">
    <normalon>../../.designer/backup/Bitcoin.svg.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>329</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>246</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="img1">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="img2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="currency1">
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EUR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KZT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="img3">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="img5">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="img4">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="currency2">
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EUR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KZT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="currency3">
       <item>
        <property name="text">
         <string>USD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EUR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KZT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="currency4">
       <item>
        <property name="text">
         <string>EUR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KZT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="currency5">
       <item>
        <property name="text">
         <string>RUB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EUR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KZT</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевести</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="add_curr"/>
    <addaction name="reset_curr"/>
    <addaction name="refresh_rate"/>
    <addaction name="separator"/>
    <addaction name="exit_btn"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add_curr">
   <property name="text">
    <string>Добавить Валюту</string>
   </property>
  </action>
  <action name="refresh_rate">
   <property name="text">
    <string>Обновить курс</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="exit_btn">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="reset_curr">
   <property name="text">
    <string>Сбросить валюты</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
